### PR TITLE
Cilium templater

### PR DESCRIPTION
**Pull request by g-gaston**
*Description of changes:*
Cilium templater to generate manifests with helm

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/networking/cilium/templater.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	}
+}
+
+func (c *Templater) GenerateUpgradePreflightManifest(ctx context.Context, spec *cluster.Spec) ([]byte, error) {
+	v := templateValues(spec)
+	v.set(true, "preflight", "enabled")
+	v.set(false, "agent")
+	v.set(false, "operator", "enabled")
+
+	manifest, err := c.helm.Template(ctx, "placeholder", spec.VersionsBundle.Cilium.Version, namespace, v) // TODO: use real oci uri
```
Comment (g-gaston) [inline pkg/networking/cilium/templater.go:32]: This is temporary until we have this info in the bundle

**Review by danbudris — APPROVED** [review]

File: pkg/networking/cilium/templater.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package cilium
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+)
+
+const namespace = "kube-system"
```
Comment (danbudris) [inline pkg/networking/cilium/templater.go:10]: oh same deal here use the constant from the constants pkg

Comment (danbudris) [inline pkg/networking/cilium/templater.go:10]:
```suggestion
const namespace = constants.KubeSystemNamespace
```

**Pull request merged**